modules/exploits/windows/http/smartermail_rce.rb

```diff
@@ -0,0 +1,203 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
```

Comment:
I believe this line should be `https://` because msftidy is complaining about it